.github/workflows/daily_ci.yml

```diff
@@ -67,3 +67,4 @@ jobs:
     uses: ./.github/workflows/library_interop_tests.yml
     with:
       dafny: ${{needs.getVersion.outputs.version}}
+      fuzz-testing: true
```

Comment:
We don't want to remove current test vector but want to add fuzz testing. This means there will be fuzz testing only